news/2018-01-20-beatmap-spotlights-december-2017.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: "Beatmap Spotlights: December 2017"
+date: 2018-01-20 09:00:00 +0000
+---
+
+It seems like we still have a few unopened presents from last year - the December Spotlights are out, bundled with a surprise announcement! Whether you are a player or mapper, make sure to drop by and check it out!
+
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+The beatmap charts for the December Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=MONTH1801) Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access. **The December 2017 Spotlights rankings will end on 15th February 2018.**
+
+## Past Spotlights results
+
+First of all, congratulations to the winners of the November 2017 Spotlights. They receive one month of supporter status for their efforts!
++ **osu!:** [Toy](https://osu.ppy.sh/users/2757689)
++ **osu!taiko:** [iceOC](https://osu.ppy.sh/users/5482401)
++ **osu!catch:** [monstratorfull](https://osu.ppy.sh/users/1646397)
++ **osu!mania:** [\_SkyFall](https://osu.ppy.sh/users/1872276)
+
+# Announcing Spotlights Bounty
+
+I'm glad to announce a new type of competition which will run simultaneous with the monthly other Spotlights. We place a bounty on a theme which lacks mapsets and then we ask you as a mapper to map a mapset and rank it until the deadline. After that we choose the bests from this pool of mapsets and announce it.
+
+## How can you enter?
+
++ Map a mapset for any Loki/Thaehan song. It doesn't matter if its not listed [here](https://osu.ppy.sh/beatmaps/artists/7) feel free to check out his other works at [bandcamp](https://loki2.bandcamp.com/) too.
```

Comment:
for any Loki/Thaehan song -> for any song by Loki/Thaehan (just reads better)
its -> it's
here -> on the featured artist page or something (featured artist page wasn't mentioned)